nixos/modules/services/web-apps/lasuite-meet.nix

```diff
@@ -0,0 +1,443 @@
+          };
+
+          DB_HOST = mkOption {
+            type = types.nullOr types.str;
+            default = if cfg.postgresql.createLocally then "/run/postgresql" else null;
+            description = "Host of the database";
+          };
+
+          REDIS_URL = mkOption {
+            type = types.nullOr types.str;
+            default =
+              if cfg.redis.createLocally then
+                "unix://${config.services.redis.servers.lasuite-meet.unixSocket}?db=1"
+              else
+                null;
+            description = "URL of the redis backend";
+          };
+
+          CELERY_BROKER_URL = mkOption {
+            type = types.nullOr types.str;
+            default =
+              if cfg.redis.createLocally then
+                "redis+socket://${config.services.redis.servers.lasuite-meet.unixSocket}?db=2"
+              else
+                null;
+            description = "URL of the redis backend for celery";
+          };
+
+          LIVEKIT_API_URL = mkOption {
+            type = types.nullOr types.str;
+            default =
+              if cfg.enableNginx && cfg.livekit.enable then
+                "http://${cfg.domain}/livekit"
+              else
+                null;
+            defaultText = lib.literalExpression ''
+              if cfg.enableNginx && cfg.livekit.enable then
+                "http://$${cfg.domain}/livekit"
+              else
+                null
+            '';
+            description = "URL to the livekit server";
+          };
+        };
+      };
+
+      default = { };
+      example = ''
+        {
+          DJANGO_ALLOWED_HOSTS = "*";
+        }
+      '';
+      description = ''
+        Configuration options of meet.
+        See https://github.com/suitenumerique/meet/blob/v${cfg.backendPackage.version}/docs/env.md
+        `REDIS_URL` and `CELERY_BROKER_URL` are set if `services.lasuite-meet.redis.createLocally` is true.
+        `DB_NAME` `DB_USER` and `DB_HOST` are set if `services.lasuite-meet.postgresql.createLocally` is true.
+      '';
+    };
+
+    environmentFile = mkOption {
+      type = types.nullOr types.path;
+      default = null;
+      description = ''
+        Path to environment file.
+
+        This can be useful to pass secrets to meet via tools like `agenix` or `sops`.
+      '';
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.lasuite-meet = {
+      description = "Meet from SuiteNumérique";
+      after =
+        [ "network.target" ]
+        ++ (optional cfg.postgresql.createLocally "postgresql.service")
+        ++ (optional cfg.redis.createLocally "redis-lasuite-meet.service");
+
+      wants =
+        (optional cfg.postgresql.createLocally "postgresql.service")
+        ++ (optional cfg.redis.createLocally "redis-lasuite-meet.service");
+
+      wantedBy = [ "multi-user.target" ];
+
+      preStart = ''
+        ln -sf ${cfg.backendPackage}/share/static /var/lib/lasuite-meet/static
```

Comment:
ln: failed to create symbolic link '/var/lib/lasuite-meet/static/static': Read-only file system